pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -174,9 +177,19 @@ func (s *Service) RegisterOperatorsRouter() {
 	router.GET("/records", getOperatorRecords)
 }
 
+// RegisterStoresRouter registers the router of the stores handler.
+func (s *Service) RegisterStoresRouter() {
+	router := s.root.Group("stores")
+	router.GET("", getAllStores)
+	router.GET("/:id", getStoreByID)
+}
+
 // RegisterRegionsRouter registers the router of the regions handler.
 func (s *Service) RegisterRegionsRouter() {
 	router := s.root.Group("regions")
+	router.GET("", getAllRegions)
+	router.GET("/:id", getRegionByID)
```

Comment:
Will we need to support GetRegionByKey in the future? If so, I think it's best to change the path to be consistent with api server?

Reply:
I think the interface may not be supported. The main purpose of this pr is to determine if the region and store are successfully synchronized between the api server and the scheduling server. cc @rleungx

Reply:
rest LGTM

Reply:
The main purpose of this PR is to debug. If we do need it later, we can add it.